src/middlewared/middlewared/etc_files/sysctl_config.py

```diff
@@ -0,0 +1,43 @@
+import os
+import subprocess
+
+
+def set_autotune_sysctl(middleware):
+    autotune_path = '/usr/local/bin/autotune'
+    if os.access(autotune_path, os.X_OK):
```

Comment:
Why os.access?

Reply:
This is to make sure that the `autotune_path` is executable

Reply:
Why would it not be?

Reply:
Copying the behaviour as it was before :) I can have it removed :)

Reply:
Yes please

Reply:
Perfect, thank you :)